core/templates/components/question-directives/question-editor/question-editor.component.ts

```diff
@@ -196,7 +199,10 @@ export class QuestionEditorComponent implements OnInit, OnDestroy {
     }
     this.solutionValidityService.init(['question']);
     const stateData = this.questionStateData;
-    stateData.interaction.defaultOutcome.setDestination(null);
+    const outcome = stateData.interaction.defaultOutcome;
+    if (outcome) {
```

Comment:
Hi @JeeveshGarg -- you should probably be aware that this change has caused a significant regression in the contributor dashboard (#16973) that has completely blocked question reviewers from editing questions. This is because this code assigns a string value to outcome.dest, but question answer groups are not supposed to have destinations since they are not part of an exploration. 

Could you please explain why you changed this to `this.questionId` instead of `null`, and what sort of testing you did for this component?